test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -471,6 +471,170 @@ def test_tensor_index_advanced_indexing(self):
     def test_tensor_index_advanced_indexing_consecutive(self):
         self._test_index_generic(lambda input: input[:, torch.tensor([0, 2]), torch.tensor([[1, 3], [4, 0]]), None])
 
+    @skipIfUnsupportedMinOpsetVersion(11)
```

Comment:
can we add a test with negative dims?

Reply:
Nice catch, currently onnx::ScatterND doesn't support negative indices. So cases with negative dims are not supported. I'll add this into the description.